### PR TITLE
Connor updated readme.md

### PR DESCRIPTION
**Pull request by cdoherty25**
I added my username

## Objective

_Note why you're making this change and any relevant issues_

## Changes made

_Note what you actual changes you're proposing_ 

## Who needs to review this Pull Request? 

_@mention some teammates who will check your Pull Request for any mistakes._
_Teammates, leave comments in a review if you approve this change_

### CODE REVIEW

**Comment by SealaneC** [comment]
This is Charles, I approve the change, please approve mine :)

**Comment by pamchansan314** [comment]
This is Pam, I approve the change.

File: 2024_Spring/readme.md [hunk 1]
```diff
@@ -25,15 +25,15 @@ Make the Pull Request against changing the file in _my repo_ (not your fork). I
 
 - [ ] Blackwell, Kelly
 
-- [X] Chansangavej, Pam (@pamchansan314) 🤿🍦
+- [ ] Chansangavej, Pam
```
Comment (joewadcan) [inline 2024_Spring/readme.md:28]: this second change was made in the PR, so can't approve as is

Comment (pamchansan314) [reply 2024_Spring/readme.md:28]: Hi, Joe. I don't quite understand what you mean.

**Comment by jainaman93** [comment]
@cdoherty25: please review the format

**Pull request closed without merging**